docs/extend/modular-resources/configure.md

```diff
@@ -8,53 +8,87 @@ description: "Add and configure a module from the Viam Registry on your robot."
+If the module requires you to configure specific **Attributes**, click the **URL** link in the [module's configuration pane](#configure-a-module-from-the-viam-registry) to view the specific attribute requirements on the module's GitHub page.
 
 To delete a module added from the Viam Registry, click the trash can icon in the upper-right corner of the module configuration pane in the **Components** tab.
+Deleting a module *does not* delete any configured modular resources it provides.
 
-### Configure a module from the Viam Registry
+### Add a modular service from the Viam registry
 
-Once you have added a module from the Viam Registry, you can view and configure the module from the **Modules** subtab:
+To add a modular [service](/services/) from the Viam Registry to your robot:
+
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Services** subtab and click the **Create service** button.
+1. Browse the list of available service types and select the specific modular service you'd like to add.
+
+   {{<imgproc src="extend/modular-resources/configure/add-service-by-category.png" resize="400x" declaredimensions=true alt="The add a component modal showing results for the mlmodelservice triton module when searching by the category 'ML Model'">}}
+
+   You can also start typing to search for a module by name or to narrow down your search results.
+
+   {{<imgproc src="extend/modular-resources/configure/add-service-by-name.png" resize="400x" declaredimensions=true alt="The add a component modal showing results for the mlmodelservice triton module when searching by the name 'triton'">}}
+
+1. After selecting the modular service, click the **Add module** button, enter a name for your modular service, and click **Create** to add it to your robot.
+
+   {{<imgproc src="extend/modular-resources/configure/add-service-screen.png" resize="400x" declaredimensions=true alt="The add a component modal showing the mlmodelservice triton module pane, with the 'Add module' button shown">}}
+
+When you add a module from the Viam Registry, the custom modular service it provides appears under the **Services** subtab like any other service.
+You can also find [the module itself](#configure-a-module-from-the-viam-registry) listed as **Deployed** under the **Modules** subtab.
+
+If the module requires you to configure specific **Attributes**, click the **URL** link in the [module's configuration pane](#configure-a-module-from-the-viam-registry) to view the specific attribute requirements on the module's GitHub page.
+
+To delete a module added from the Viam Registry, click the trash can icon in the upper-right corner of the module configuration pane in the **Services** tab.
+Deleting a module *does not* delete any configured modular resources it provides.
```

Comment:
Good call out. We'll need to update this particular line when deleting the module _does_ delete those resources, though the UI will make it obvious